mkdocs.yml

```diff
@@ -399,6 +399,7 @@ plugins:
               "Chart/configuration/integrations/email.md": "chart/configuration/integrations/email.md"
               "Chart/configuration/tls-ingress.md": "index.md"
               "Chart/configuration/monitoring.md": "chart/configuration/monitoring.md"
+              "Chart/configuration/logging.md": "chart/configuration/logging.md"
```

Comment:
We don't really need this line, these are just to redirect some old URLs from a time when "Chart" was capitalized. :nerd_face: 

```suggestion
```